### PR TITLE
PKG-1613 - Update to 67.8.0

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 67.8.0
- Verified no additional patches were required on distutils (compared to what is applied on python)
- Add entry_points
- Add distutils as imports

https://github.com/pypa/setuptools/tree/v67.8.0
https://github.com/pypa/setuptools/blob/v67.8.0/CHANGES.rst

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,6 +20,61 @@ source:
 build:
   number: 0
   skip: True               # [py<37]
+  entry_points:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:23]: Usually, we add only `console_scripts`. I'm not 100% sure if we want all other types of entry_points

**Review by skupr-anaconda — APPROVED** [review]
LGTM but entry_points

File: recipe/meta.yaml (hunk 1 above)
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:23]: just out of curiosity how did know about those entry points ?

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 2]
```diff
@@ -20,6 +20,61 @@ source:
...
+    # distutils.commands
+    - alias=setuptools.command.alias:alias
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:25]: I think only [console_scripts](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/?highlight=console_scripts#console-scripts) have to be listed as entry points here.

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:25]: Yeah, see https://github.com/conda/conda-build/blob/78d2a4758be9a5b2e6c194b562ae6bd16092ade6/conda_build/utils.py#L1122. In conda, they are used to create console scripts. So all these will pollute the environment with random commands.

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]
See https://github.com/AnacondaRecipes/setuptools-feedstock/pull/56/files#r1209389634

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml]: Thanks, indeed that's not needed. I removed them.

Comment (cbouss) [reply recipe/meta.yaml]: Bad addition. I removed them.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**